api-common/src/main/kotlin/io/qalipsis/api/meters/CampaignMeterRegistry.kt

```diff
@@ -123,6 +126,59 @@ interface CampaignMeterRegistry {
      */
     fun counter(name: String, vararg tags: String): Counter
 
+    /**
+     * Creates a new [Rate] metric to be added to the registry. This metric calculates the
+     * ratio between two independently tracked [Gauge] metrics.
+     *
+     * @param scenarioName the name of the scenario under which the rate is collected
+     * @param stepName the name of a step within the scenario
+     * @param name the name of the metric
+     * @param tags additional key-value pairs to associate with the rate metric
+     *
+     * @sample rateExample
+     */
+    fun rate(
+        scenarioName: ScenarioName,
+        stepName: StepName,
+        name: String,
+        tags: Map<String, String> = emptyMap(),
+    ): Rate
+
+    /**
+     * Creates a new [Throughput] metric to be added to the registry. This metric
+     * tracks the number of hits measured per a configured unit of time, default to seconds.
+     *
+     * @param scenarioName the name of the scenario within which the throughput is measured
+     * @param stepName the name of a step within the scenario
+     * @param name the name of the metric
+     * @param tags additional key-value pairs to associate with the rate metric
+     * @param measurementInterval the duration over which an operation is measured. It determines the duration of time
+     * between each throughput calculation, defaults to seconds.
+     * @param percentiles a list of values within the range of 1.0-100.0, representing specific points of observation,
+     * defaults to an empty list
+     *
+     * @sample throughputExample
+     */
+    fun throughput(
+        scenarioName: ScenarioName,
+        stepName: StepName,
+        name: String,
+        measurementInterval: Duration,
```

Comment:
Let's use a TimeUnit instead. Duration is not really fancy to use... And this will probably make more sense for the users.

Set TimeUnit.SECOND as default.